src/app/components/summary/summary.component.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SummaryComponent } from './summary.component';
+import {FormsModule} from "@angular/forms";
+import {ActivatedRoute, RouterModule} from "@angular/router";
+import {MaterialModule} from "../../material.module";
+import {BrowserAnimationsModule} from "@angular/platform-browser/animations";
+import {BrowserModule} from "@angular/platform-browser";
+import {RoomService} from "../../services/room.service";
+import {HttpClientModule} from "@angular/common/http";
+import {CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA} from "@angular/core";
+import {MatDialog, MatToolbar} from "@angular/material";
+import {RouterTestingModule} from "@angular/router/testing";
+
+describe('SummaryComponent', () => {
+  let component: SummaryComponent;
```

Comment:
The `component`'s value does not change in the tests, right? If so, consider using `const` insteaf of `let`

Reply:
I will check this